yapi.py

```diff
@@ -38,44 +39,62 @@
 no_answer = ("N", "No", "n", "no")
 right_answer = yes_answer + no_answer
 
-continue_to_ask = True
 
-while continue_to_ask:
-    print("-" * 79)
-    print("You can choose to install this packages:")
-    for package_counter in packages:
-        print("{:>2}) {} - {}".format(
-            package_counter,
-            packages[package_counter][0],
-            packages[package_counter][1]))
-    choose = -1
-    while choose not in range(0, package_counter + 1):
-        choose = int(input("What package do you want to install? "))
-    package_to_install = choose
-    while choose not in right_answer:
-        choose = input(
-            "Do you want to install {}? ".format(
-                packages[package_to_install][0]))
-    if choose in yes_answer:
+
+if len(sys.argv) == 2:
+    print("You must provide another argument for the package or no arguments for the question installer")
+    print("python yapi.py install package for example")
+if len(sys.argv) == 3:
```

Comment:
A truth value of an if excludes the other, so it is not necessary to control them all. An elif it's better because you save performances, not really many but it is still worth keeping down the running costs.